package.json

```diff
@@ -32,12 +32,18 @@
     "eslint-plugin-prettier": "^3.4.0",
     "eslint-plugin-react": "^7.23.2",
     "eslint-plugin-react-hooks": "^4.2.0",
+    "netlify-plugin-cache": "^1.0.3",
     "prettier": "^2.2.1"
   },
   "resolutions": {
     "react-scripts": "patch:react-scripts@npm:4.0.2#.yarn/patches/react-scripts@npm:4.0.2.patch",
     "react-redux": "npm:7.2.4",
     "react": "npm:17.0.2",
     "react-dom": "npm:17.0.2"
+  },
+  "scripts": {
+    "build:examples": "yarn workspaces foreach --include '@reduxjs/*' --include '@examples-query-react/*' -vtp run build",
+    "build:docs": "yarn workspace docs run build",
+    "build:package": "yarn workspace @reduxjs/toolkit run build"
```

Comment:
Fun fact: scripts containing `:` are global so you could define these two in their respective workspaces and you can call them from anywhere

Reply:
Uuuh, TIL. Thanks!
But I think for visibility we'll keep them there. Personally I just always look into the package.json what scripts are available

Reply:
Understandable, another fun fact: `yarn run` will print all the scripts you can run, though I see it doesn't include global scripts from other workspaces yet